### DISCUSSION TITLE
Help, NWMe 10 hours

### DISCUSSION THREAD

**Discussion by NicNem55** [Chia-Network/chia-blockchain#2311] [Q&A]
Seeding speed k = 32 / 101.4GB in 10 hours, what's wrong?

Come back wallet as 1.0.5, Windows 10,
Intel Xeon E5-2678 v3 @ 2.50GHz 12 cores 24 threads, 32 GB RAM, the system is installed on a separate 250 GB NWMe disk, seeding occurs on a 1 TB NWMe disk, storage on the HDD.
The setting is as follows, https://prnt.sc/11tp96a, I launch 3 sections.
Оr is this the normal sowing time?

**Comment by cyr616** [comment]
your cpu single thread performance is poor.

**Reply by NicNem55** [reply]
> 
> 
> производительность вашего процессора в однопоточном режиме оставляет желать лучшего.

2678 V3

**Reply by cyr616** [reply]
year I know,this cpu single thread performance score in cpu-z is about 360 points. the one with 500 points can inprove to about 7 hours. I think you can upgrade your SSD to 2TB and start 7-8 sections in the same time.

**Reply by NicNem55** [reply]
is there a table by points?
